keras/engine/training_arrays.py

```diff
@@ -32,7 +32,8 @@ def fit_loop(model, fit_function, fit_inputs,
              initial_epoch=0,
              steps_per_epoch=None,
              validation_steps=None,
-             validation_freq=1):
+             validation_freq=1,
+             seed=113):
```

Comment:
We should no use default seeds. This would dramatically reduce randomness (effectively remove randomness since there is no user-facing way to set the seed).